### PR TITLE
feat(Tooltip): add left align option

### PR DESCRIPTION
**Pull request by fgameiro**
affects: @crave/farmblocks-tooltip

ISSUES CLOSED: #124

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -12,25 +12,31 @@
     "test": "jest",
```
Comment (fczuardi) [inline package.json]: the diff is showing formatting changes, is pre-commit configured?

File: packages/tooltip/src/styledComponents/Tooltip.js [hunk 2]
```diff
@@ -33,14 +37,14 @@ const StyledTooltip = styled.div`
...
   }
 
   &:after {
     border-color: transparent;
     border-bottom-color: #ffffff;
     border-width: 7px;
-    right: 8px;
+    ${props => props.align}: 8px;
   }
 `;
```
Comment (fczuardi) [inline packages/tooltip/src/styledComponents/Tooltip.js]: 👍

File: package-lock.json [hunk 3]
```diff
@@ -0,0 +1,15253 @@
+{
```
Comment (fczuardi) [inline package-lock.json]: please delete this file, we are using yarn.lock to freeze our dependencies. Feel free to add it to the .gitignore file as well..

File: packages/tooltip/src/Tooltip.js [hunk 4]
```diff
@@ -14,11 +14,13 @@ const Tooltip = (props: Object) => (
 
 Tooltip.propTypes = {
   text: PropTypes.string.isRequired,
-  isVisible: PropTypes.bool
+  isVisible: PropTypes.bool,
```
Comment (fczuardi) [inline packages/tooltip/src/Tooltip.js]: I have opened issue #126 for us to refactor this property name

File: packages/tooltip/src/styledComponents/Tooltip.js [hunk 5]
```diff
@@ -1,7 +1,10 @@
 // @flow
 import styled from "styled-components";
+import { colors } from "@crave/farmblocks-theme";
 
-const Container = styled.div`position: relative;`;
```
Comment (fczuardi) [inline packages/tooltip/src/styledComponents/Tooltip.js]: same as above, non prettiefied code shouldnt be commited, a proper installed pre-commit should fix this

File: package-lock.json (hunk 3 above)
Comment (fgameiro) [reply package-lock.json]: Ok!

File: package.json (hunk 1 above)
Comment (fgameiro) [reply package.json]: It wasn't. I'm going to activate it and reformat this file.

File: packages/tooltip/src/styledComponents/Tooltip.js (hunk 5 above)
Comment (fgameiro) [reply packages/tooltip/src/styledComponents/Tooltip.js]: Indeed pre-commit fixed this.

**Pull request merged**